site2/docs/admin-api-overview.md

```diff
@@ -9,15 +9,26 @@ The Pulsar admin interface enables you to manage all important entities in a Pul
 You can interact with the admin interface via:
 
 - HTTP calls, which are made against the admin {@inject: rest:REST:/} API provided by Pulsar brokers. For some RESTful APIs, they might be redirected to the owner brokers for serving with [`307 Temporary Redirect`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307), hence the HTTP callers should handle `307 Temporary Redirect`. If you use `curl` commands, you should specify `-L` to handle redirections.
+  
+  > **Tip**
```

Comment:
Why is this a "tip" and on the other page it's "important"?

Reply:
Updated these "tips" to "important" for keeping consistent.